### PR TITLE
Enable reproducible builds

### PR DESCRIPTION
**Pull request by meziantou**

### CODE REVIEW

File: Directory.Build.props [hunk 1]
```diff
@@ -1,35 +1,36 @@
...
+
     <!-- Compile Options -->
     <LangVersion>9.0</LangVersion>
     <Deterministic>true</Deterministic>
     <Features>strict</Features>
     <TreatWarningsAsErrors Condition="'$(Configuration)' != 'Debug'">true</TreatWarningsAsErrors>
 
     <RestoreUseStaticGraphEvaluation>true</RestoreUseStaticGraphEvaluation>
+	
+	<!-- Package info -->
```
Comment (louis-z) [inline Directory.Build.props]: Replace TABs with spaces throughout the file.

File: Directory.Build.props [hunk 2]
```diff
@@ -1,35 +1,36 @@
...
   </PropertyGroup>
+  
+  <ItemGroup>  
+    <None Include="$(MSBuildThisFileDirectory)\LICENSE" Pack="true" PackagePath="" Visible="false" />
+    <None Include="$(MSBuildThisFileDirectory)\README.md" Pack="true" PackagePath="" />
+	
+    <SourceRoot Include="$(MSBuildThisFileDirectory)/" />
 
-  <!-- SourceLink: https://www.meziantou.net/how-to-debug-nuget-packages-using-sourcelink.htm -->
-  <PropertyGroup Condition="'$(_IsGitHub)' == 'true'">
```
Comment (louis-z) [inline Directory.Build.props:22]: How come this whole section is gone?

File: Directory.Build.props [hunk 3]
```diff
@@ -1,35 +1,36 @@
...
-    <PublishRepositoryUrl>true</PublishRepositoryUrl>
-    <EmbedUntrackedSources>true</EmbedUntrackedSources>
-    <IncludeSymbols>true</IncludeSymbols>
-    <SymbolPackageFormat>snupkg</SymbolPackageFormat>
-    <ContinuousIntegrationBuild>$(_ContinuousIntegrationBuild)</ContinuousIntegrationBuild>
-  </PropertyGroup>
+	<PackageReference Include="MinVer" PrivateAssets="All" Version="2.5.0" />
 
-  <ItemGroup Condition="'$(_IsGitHub)' == 'true'">
-    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0-*" PrivateAssets="All"/>
```
Comment (louis-z) [inline Directory.Build.props:31]: Why do we no longer use `Microsoft.SourceLink.GitHub`?

**Review by louis-z — CHANGES_REQUESTED** [review]

File: Directory.Build.props (hunk 2 above)
Comment (meziantou) [reply Directory.Build.props:22]: The package `DotNet.ReproducibleBuilds` includes everything :)

You can check the props/target files in https://github.com/dotnet/reproducible-builds/tree/main/src/DotNet.ReproducibleBuilds

File: Directory.Build.props (hunk 3 above)
Comment (meziantou) [reply Directory.Build.props:31]: The package `DotNet.ReproducibleBuilds` includes everything, including a dependency on `Microsoft.SourceLink.GitHub` :)

You can check the nuspec file in https://github.com/dotnet/reproducible-builds/tree/main/src/DotNet.ReproducibleBuilds

File: Directory.Build.props (hunk 1 above)
Comment (meziantou) [reply Directory.Build.props]: done

**Review by louis-z — APPROVED** [review]

**Pull request merged**